src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
> If users can't trust that their settings will be saved and that all upgrades are backwards compatible then they will lose faith in Mixxx's technical design.

They're not "settings". The user has no explicit interaction with these state variables.... Having the window position reset to the default position once in 5 years is hardly going to ruin a DJs career lol. I still don't see where you're imagining this maintenance burden to be... In general, more code = more bugs = more maintenance burden IMHO. It doesn't look like we're going to come to an agreement on this point though, so I guess we move on.

> Hard to say what you should try next without some more info.

I was getting a compile error... `toBase64()` returns a `QByteArray` and there was no method that takes a `QByteArray`. I can post more info tomorrow if necessary.